content/blog/2018-01-23-tidyhydat.md

```diff
@@ -0,0 +1,215 @@
+---
+slug: tidyhydat
+title: "5 Things I Learned Making a Package to Work with Hydrometric Data in R"
+package_version: 0.3.2
+authors:
+- name: Sam Albers
+  url: https://github.com/bcgov
+date: 2018-01-23
+categories: blog
+topicid:
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- hydrometrics
+- bcgov
+- tidydata
+- Canada
+- climate-data
+- data-access
+- earth-science
+output:
+  html_document:
+    keep_md: yes
+---
+
+
+
+One of the best things about learning R is that no matter your skill level, there is always someone who can benefit from your experience. Topics in R ranging from complicated machine learning approaches to calculating a mean all find their relevant audiences. This is particularly true when writing R packages. With an ever evolving R package development landscape (R, GitHub, external data, CRAN, continuous integration, users), there is a strong possibility that you will be taken into regions of the R world that you never knew existed. More experienced developers may not get stuck in these regions and therefore not think to shine a light on them. It is the objective of this post to explore some of those regions in the R world that were [highlighted for me when the `tidyhydat` package was reviewed by rOpenSci](https://github.com/ropensci/onboarding/issues/152). 
+
+`tidyhydat` is a new rOpenSci R package that provides a standard method of accessing [Environment and Climate Change Canada's](https://www.canada.ca/en/environment-climate-change.html) [Water Survey of Canada](https://wateroffice.ec.gc.ca/index_e.html) data sources (WSC) using a consistent and easy to use interface that employs [tidy data principles](https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html). I developed the package because I was repeating the [same steps](https://twitter.com/drob/status/611885584584441856?ref_src=twsrc%5Etfw&ref_url=http%3A%2F%2Ftinyheero.github.io%2Fjekyll%2Fupdate%2F2015%2F07%2F26%2Fmaking-your-first-R-package.html) to [import and tidy](http://r4ds.had.co.nz/explore-intro.html) Water Survey of Canada data in R. Many useRs will read this post and think "I’d know how to do that". To you I tip my hat. To those who read this and think "huh, never knew that", you are the target audience. In this post, I'll focus on 5 things that I learned  during the `tidyhydat` rOpenSci review, provide a brief introduction on how the package works and provide an example use case. 
```

Comment:
you spell out "Water Survey of Canada" after defining abbreviation. decide which to do throughout post